src/test/unit/profile-tree.test.js

```diff
@@ -32,8 +29,19 @@ describe('unfiltered call tree', function() {
   const H = 7;
   const I = 8;
 
+  function getProfile() {
+    return getProfileFromTextSamples(`
+      A A A
+      B B B
+      C C H
+      D F I
+      E G
+    `).profile;
+  }
```

Comment:
This is approximately what I suggest above :) Except returning a function from `getProfileFromTextSamples` would force this construction.